src/mono/wasm/debugger/tests/Directory.Build.props

```diff
@@ -16,4 +16,9 @@
 
     <MicrosoftNetCoreAppRuntimePackRidDir>$(ArtifactsBinDir)microsoft.netcore.app.runtime.browser-wasm\$(RuntimeConfiguration)\runtimes\browser-wasm\</MicrosoftNetCoreAppRuntimePackRidDir>
   </PropertyGroup>
+  
+  <PropertyGroup Condition="'$(RuntimeConfiguration)'=='Debug'">
+    <WasmDebugLevel>5</WasmDebugLevel>
+  </PropertyGroup>
```

Comment:
Um this seems excessive to have by default. It makes the test output large, and difficult to read.

IMHO, whoever is running the tests can set it from the command line, eg. `make run-debugger-tests MSBUILD_ARGS="/p:WasmDebugLevel=5"`.

Reply:
removed and added into readme